book/src/users/startup.md

```diff
@@ -0,0 +1,65 @@
+# Startup
+
+Configuration occurs at startup via flags:
+
+```
+cargo run -p trin -- --flag1 flag1value --flag2 flag2value
```

Comment:
`cargo run -p trin -- --flag1 value --flag2 value` and same for the command below